kolibri/core/assets/src/vue/core-modal/index.vue

```diff
@@ -109,6 +102,9 @@
       closeModal() {
         this.$dispatch('close');
       },
+      submitModal() {
+        this.$dispatch('submit');
+      },
```

Comment:
I'm going to merge in so that I can move forward on using this for import/export. However let's plan on discussing the API of the modal component when we're both back in the office.